eox_nelp/signals/receivers.py

```diff
@@ -355,9 +355,12 @@ def mt_course_passed_handler(user, course_id, **kwargs):  # pylint: disable=unus
     if not getattr(settings, "ACTIVATE_MT_TRAINING_STAGE", False):
         return
 
+    extra_info = getattr(user, "extrainfo", None)
+    national_id = extra_info.national_id if extra_info else user.username
```

Comment:
What about if national_id is "". that would be sent?. Is weird to found same usernames. But this this field of extra_info seems allow blank